src/panels/config/scene/ha-scene-editor.ts

```diff
@@ -0,0 +1,727 @@
+import {
+  LitElement,
+  TemplateResult,
+  html,
+  CSSResult,
+  css,
+  PropertyValues,
+  property,
+  customElement,
+} from "lit-element";
+import "@polymer/app-layout/app-header/app-header";
+import "@polymer/app-layout/app-toolbar/app-toolbar";
+import "@polymer/paper-icon-button/paper-icon-button";
+import "@polymer/paper-item/paper-item";
+import "@polymer/paper-item/paper-icon-item";
+import "@polymer/paper-item/paper-item-body";
+
+import { classMap } from "lit-html/directives/class-map";
+
+import "../../../components/ha-fab";
+import "../../../components/device/ha-device-picker";
+import "../../../components/entity/ha-entities-picker";
+import "../../../components/ha-paper-icon-button-arrow-prev";
+import "../../../layouts/ha-app-layout";
+
+import { computeStateName } from "../../../common/entity/compute_state_name";
+
+import { haStyle } from "../../../resources/styles";
+import { HomeAssistant } from "../../../types";
+import { navigate } from "../../../common/navigate";
+import { computeRTL } from "../../../common/util/compute_rtl";
+import {
+  SceneEntity,
+  SceneConfig,
+  getSceneConfig,
+  deleteScene,
+  saveScene,
+  IGNORED_DOMAINS,
```

Comment:
Because auto complete is throughout all exported constants, I'm a fan of making the name descriptive on its own. So `SCENE_IGNORED_DOMAINS`